src/test/groovy/at/bxm/gradleplugins/svntools/SvnToolsPluginTest.groovy

```diff
@@ -9,4 +11,28 @@ class SvnToolsPluginTest extends SvnTestSupport {
     then: "extension is defined"
     project.extensions.getByType(SvnToolsPluginExtension)
   }
+
+  def "Tasks are available by name"(String name, Class<?> taskClass) {
+    expect:
+    def project = projectWithPlugin()
+    def task = project.task(type: project.extensions.extraProperties.get(name), name.toLowerCase())
```

Comment:
I think you should write something like `type: project."$name"` instead of `type: project.extensions.extraProperties.get(name)` to actually test this feature ;-)